sdks/python/apache_beam/metrics/cells.py

```diff
@@ -31,7 +31,9 @@
 
 
 class CellCommitState(object):
```

Comment:
Seems like lint was crashing due to this change (empty __all__ tag) for some reason. So had to revert this.